hc/api/models.py

```diff
@@ -145,18 +145,13 @@ class Ping(models.Model):
     method = models.CharField(max_length=10, blank=True)
     ua = models.CharField(max_length=200, blank=True)
 
-class Report(models.Model):
```

Comment:
You seem to have deleted a model from the code base. Confirm it was necessary? And does not affect some parts of it.

Reply:
This model was no longer necessary in the report implementation and needed to be removed